CHANGELOG.md

```
@@ -1,3 +1,358 @@
+# 1.0.0 (2026-03-19)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)